mmdeploy/mmcv/ops/nms.py

```diff
@@ -345,7 +345,8 @@ def _multiclass_nms_single(boxes: Tensor,
     labels = labels[:, topk_inds, ...]
 
     if output_index:
-        bbox_index = pre_topk_inds[None, topk_inds]
+        bbox_index = pre_topk_inds[box_inds]
+        bbox_index = bbox_index[None, topk_inds[:-1]]
```

Comment:
what's `[:-1]` for?

Reply:
https://github.com/open-mmlab/mmdeploy/blob/90f17f2e62ef3079dfcdb21c364892a43a2ff210/mmdeploy/mmcv/ops/nms.py#L332-L334
There's a pad here, would be out of bound without [:-1].